.github/workflows/performance.yml

```diff
@@ -53,7 +51,7 @@ jobs:
   # - Ensures version-controlled files are not modified or deleted.
   performance:
     runs-on: ubuntu-latest
-    if: ${{ github.repository == '10up/wordpress-develop' || github.event_name == 'push' }}
+    if: ${{ github.repository == 'WordPress/wordpress-develop' || github.event_name == 'pull_request' }}
```

Comment:
should we add a conditional so this only runs when the PR has been merged, eg. if: `github.event.pull_request.merged == true`

Reply:
I don't think so, I think running the workflow before merges is useful info during a review, but we only want to log the data when the PR is merged.

Reply:
@joemcgill See my reply in https://github.com/10up/wordpress-develop/pull/41#discussion_r1119187274: There may be benefits in sending data prior to this PR getting merged, even to show how and _that_ it's working.

Reply:
This is simply following the pattern of the other workflows in the repo, I think it's useful for this to run just like other workflows, but am open to us considering other alternatives. However, I'd love to leave it as is and open this as a conversation on the Core PR so we can get feedback from folks who have been maintaining the other jobs.

Reply:
@joemcgill My comment here was not about this line of code, rather just addressing what you had said in your comment before mine :)

I think this line of code is good as is for now.